stdlib/test/collections/test_set.mojo

```diff
@@ -484,6 +485,14 @@ def test_clear():
     assert_true(len(set5) == 0)
 
 
+def test_set_str():
+    var a = Set[Int](1, 2, 3)
+    AE(a.__str__(), "{1, 2, 3}")
+    AE(a.__repr__(), "{1, 2, 3}")
+    var b = Set[String]("a", "b")
+    AE(b.__str__(), "{'a', 'b'}")
```

Comment:
Nice PR, thanks! Can you add a test case for an empty set? It seems like a common edge case.